index.html

```diff
@@ -1674,7 +1674,9 @@ <h3>Validity Period</h3>
 <a>credential</a> becomes valid, which could be a date and time in the future.
 Note that this value represents the earliest point in time
 at which the information associated with the <code>credentialSubject</code>
-<a>property</a> becomes valid.
+<a>property</a> becomes valid. If a `validUntil` value also exists, the
+`validFrom` value MUST be temporally the same or earlier than the `validUntil`
+value.
```

Comment:
@decentralgabe @David-Chadwick @msporny — What do you think? Parallel change also applied below...
```suggestion
<a>property</a> becomes valid. If a `validUntil` value also exists, the
`validFrom` value MUST express a datetime that is temporally the same or 
earlier than the datetime expressed by the `validUntil` value.
```